api/ipsets/update

```diff
@@ -26,14 +26,15 @@
 data=$(cat /dev/stdin)
 
 status=$(echo $data | jq -r '.status')
+geoipStatus=$(echo $data | jq -r '.geoipStatus')
```

Comment:
The first letter of the prop should be in upper case to mach all other props.
Please change it to `GeoipStatus`.

As a reference, I will going to propose fixes only of this file but please replace all occurrences.

```suggestion
geoipStatus=$(echo $data | jq -r '.GeoipStatus')
```

Reply:
done